src/infix-eng-rev.ts

```diff
@@ -0,0 +1,15 @@
+
+import { AbstractInfixNotation } from "./infix-abstract";
+
+export class InfixEngineeringReverseNotation extends AbstractInfixNotation {
+	public get name(): string {
+		return "Reverse infix engineering";
+	}
+
+	protected formatMantissa(digit: number):string {
```

Comment:
Codacy found an issue: [expected onespace after colon in call-signature](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)